### DISCUSSION TITLE
Windows Security keeps flagging Transmission as PUA

### DISCUSSION THREAD

**Discussion by jmadarg** [transmission/transmission#4605] [Q&A] [answered]
### What is the issue?

For some reason, Windows Security (the evolution of Defender on Windows 11)  has been flagging Transmission as a Potencially Unwanted App for like a month. I had v3.0 for a long while, and suddenly it got removed by Windows Security, didn't think much and just used the opportunity to update to v4.0, it got flagged again, but after allowing it and disabling that module (don't worry, I'm pretty careful of what I download) I managed to install it and use it.

Yesterday it got flagged again out of nowhere, I really don't know how if the Potencially Unwanted App Blocking is disabled.

![image](https://user-images.githubusercontent.com/62086445/211152735-cc6b4672-2a8e-47c1-bb41-bd8c087ce683.png)
![image](https://user-images.githubusercontent.com/62086445/211152764-4f1ca40d-000b-4175-91b9-18934060b1ca.png)

And now I can't even download the installer without it getting flagged.

![image](https://user-images.githubusercontent.com/62086445/211152806-cf9aceb0-b3e5-4780-80bb-963cd9b10fcd.png)

I know that this is probably not an issue on your end, and I managed to install it by allowing the installation msi and the program's exe, but could be a big problem for some people if they get the same false positive. Hope you are able to do something, transmission is awesome!

### Which application of Transmission?

Qt app on Windows

### Which version of Transmission?

transmission-4.0.0-beta.3

**Comment by jmadarg** [comment]
It keeps happening, I really don't understand why.

![image](https://user-images.githubusercontent.com/62086445/212824449-795bfdab-f8d0-4b2c-b327-1df67cf1651c.png)

**Comment by ckerr** [comment]
@mikedld do you know? (I don't...)

**Comment by mikedld** [comment, marked as answer]
https://forum.transmissionbt.com/viewtopic.php?p=79684#p79684

**Comment by Malumen** [comment]
Just to make sure you don't get buried by these two heads of `transmission` development:
https://github.com/transmission/transmission/issues/6227
@jmadarg a bunch of us are also getting strange actions from the app, more than just simply being flagged.

As much as devs wanna hand-wave this away, I'm glad this discussion is still open and not locked out of sight...

**Reply by ckerr** [reply]
@Malumen if you would respond in that thread and answer the questions that were asked, that would be great
